space_packet_parser/parameters.py

```diff
@@ -257,14 +257,14 @@ def parse_value(self, packet: packets.CCSDSPacket, **kwargs):
         derived_value : str
```

Comment:
Fix docstring type hinting and add type hint to function signature. This goes for all the `parse_value` methods.